crates/wasm/src/lib.rs

```diff
@@ -74,6 +77,20 @@ pub fn get_address_by_index(full_viewing_key: &str, index: u32) -> JsValue {
     return serde_wasm_bindgen::to_value(&address_str).unwrap();
 }
 
+#[wasm_bindgen]
+pub fn get_ephemeral_address(full_viewing_key: &str, index: u32) -> JsValue {
```

Comment:
`JsValue` is quite painful trying to understand what comes back from this function. We also need to start adding docstrings above every imported function so it helps the frontend more. Something like:

```rust
/// Generates a kind of "burner" address in which the user can use to maintain their anonymity without sharing their main address.
/// Returns: `u128`
#[wasm_bindgen]
pub fn get_ephemeral_address(full_viewing_key: &str, index: u32) -> JsValue {
```

and this would work for parameters too:

```rust
/// Scans block for notes, swaps...
/// Arguments: 
///     compact_block: `v1alpha1::CompactBlock`
/// Returns: `ScanBlockResult`
#[wasm_bindgen]
    pub fn scan_block_without_updates(&mut self, compact_block: JsValue) -> JsValue {
```